config/default/kustomization.yaml

```diff
@@ -204,16 +204,3 @@ patches:
 #   target:
 #     kind: CustomResourceDefinition
 #     name: inferenceservices.serving.kserve.io
-- patch: |-
```

Comment:
we can remove the commented lines above as well

Reply:
removed.